src/olympia/devhub/templates/devhub/addons/submit/sidebar.html

```diff
@@ -19,24 +19,27 @@
   </hgroup>
   <ol class="submit-addon-progress{% if addon and not addon.is_listed %} unlisted{% endif %}">
     {% for text, class in NAV %}
-      <li class="{{ class }}{% if step.current == loop.index %} current{% endif %}">
-        {% if step.current < HAS_ADDON %}
-          {% if loop.index <= step.current %}
-            <a href="{{ url(BASE % loop.index) }}">{{ text }}</a>
-          {% else %}
-            {{ text }}
-          {% endif %}
-        {% else %}
-          {# 1. We have an addon, so don't link to non-addon steps.
-             2. Don't link steps above the max step the addon has reached.
-             3. If step.max == MAX the addon is done, so we only show the final page. #}
-          {% if loop.index < HAS_ADDON or loop.index > step.max or step.max == MAX %}
-            {{ text }}
-          {% else %}
-            <a href="{{ url(BASE % loop.index, addon.slug) }}">{{ text }}</a>
-          {% endif %}
-        {% endif %}
-      </li>
+      {# Skip adding the first list item (Getting Started) if already accepted. #}
+      {% if step.current == 1 or loop.index > 1 %}
+         <li class="{{ class }}{% if step.current == loop.index %} current{% endif %}">
+           {% if step.current < HAS_ADDON %}
+             {% if loop.index <= step.current %}
```

Comment:
I think you want to update this `if` instead so the text is there but it isn't a link.